homework/hw02_db_script.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+import dataset
+
+def create_table(df, columns):
+    table = df[columns].copy()
+    table = table.drop_duplicates().reset_index(drop=True)
+    table = table.astype(object).where((pd.notnull(table)), None)
+    return table
+
+
+# EXTRACT
+data_base = dataset.connect("postgresql://postgres@localhost/data_base_hw01")
+products = pd.DataFrame(data=list(data_base['products'].all()))
+offices = pd.DataFrame(data=list(data_base['offices'].all()))
+employees = pd.DataFrame(data=list(data_base['employees'].all()))
+customers = pd.DataFrame(data=list(data_base['customers'].all()))
+orders = pd.DataFrame(data=list(data_base['orders'].all()))
+items = pd.DataFrame(data=list(data_base['items'].all()))
+
+
+# TRANSFORM
+
+## orders
+items_ = create_table(items, ['order_number', 'order_line_number'])
```

Comment:
Once you start finding yourself wanting to use the same name, it might mean you have too many things in the scope! Why not fold some of these into functions? Then they can have any local variable names they want.